airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -57,7 +57,7 @@
 - name: Amazon Ads
   sourceDefinitionId: c6b0a29e-1da9-4512-9002-7bfd0cba2246
   dockerRepository: airbyte/source-amazon-ads
-  dockerImageTag: 0.1.27
+  dockerImageTag: 0.1.28
```

Comment:
you don't need to bump version here, it's done automatically by publishing scripts